pkg/operator/certrotation/utils.go

```diff
@@ -0,0 +1,39 @@
+package certrotation
+
+import (
+	"strconv"
+	"time"
+)
+
+// durationRound formats a duration into a human-readable string.
+func durationRound(d time.Duration) string {
+	u := uint64(d)
+	neg := d < 0
+	if neg {
```

Comment:
Why not do away with `neg` and just do `if d < 0`?

Also, why do we just sign flip it if negative?

Also, why not just use `String()` of `time.Duration`?

Reply:
This is a copy of [sprig's implementation](https://github.com/gomodules/sprig/blob/master/date.go#L97-L139), which can accept negative durations (this is why the flip is required). We don't need `neg` anymore so yes, I think its worth being removed.

The issue with `time.Duration.String()` is that it returns string like `72h00m` instead of `3d` .

Added these notes as comments in the code too.